### PR TITLE
chore: remove RO user for Postgres

### PR DESCRIPTION
**Pull request by MrPink**
No longer required

### CODE REVIEW

File: src/aether-producer/templates/deployment.yaml [hunk 1]
```diff
@@ -84,14 +84,14 @@ spec:
...
-            value: {{ .Values.app.kernel.readonly_user }}
+            value: {{ .Values.app.kernel.user }}
           - name: POSTGRES_HOST
             value: {{ quote .Values.app.db.host }}
           - name: POSTGRES_PASSWORD
             valueFrom:
               secretKeyRef:
                 name: {{ .Values.app.kernel.secret }}
-                key: kernel-readonly-db-password
+                key: kernel-django-admin-password
```
Comment (obdulia-losantos) [inline src/aether-producer/templates/deployment.yaml]: `kernel-database-password` ?

File: src/aether-producer/templates/deployment.yaml [hunk 2]
```diff
@@ -84,14 +84,14 @@ spec:
           - name: PRODUCER_SETTINGS_FILE
             value:  {{ quote .Values.application.config }}
           - name: POSTGRES_USER
-            value: {{ .Values.app.kernel.readonly_user }}
+            value: {{ .Values.app.kernel.user }}
```
Comment (obdulia-losantos) [inline src/aether-producer/templates/deployment.yaml]: `kernel-database-user` ?

File: src/aether-producer/templates/deployment.yaml (hunk 1 above)
Comment (obdulia-losantos) [reply src/aether-producer/templates/deployment.yaml]: it's not the django user (not yet) but the database user :grimacing:

Comment (MrPink) [reply src/aether-producer/templates/deployment.yaml]: doh!

**Pull request merged**